src/arguments/api_test.py

```diff
@@ -42,7 +42,39 @@ def fetch_input_url(cls):
             "request_url" : request_url,
             "request_headers" : request_headers,
         }
-
+    @classmethod
+    def read_data_from_file(cls):
+        filename = input("Enter a filename (response_data.json)")
+        if filename == '':
+            print("filename empty, so default file (response_data.json) is used ")
+        with open(filename, "r") as reader:
+            file_content = reader.read()
+            try:
+                json_data = json.loads(file_content)
+                data = json_data.get("data")
+                return data
+                # Make sure the data is not None and send
+            except json.JSONDecodeError:
+                 print("Unable to parse the file, Please try again")
+                 cls.read_data_from_file()
+    @classmethod
+    def fetch_payload_data(cls):
+        store = int(input("Please choose the below options? (1/2/3)"))
+        print("Option 1: For sending data payload from terminal\n")
+        print("Option 2: For sending data payload by reading from json file\n")
+        print("Option 3: For not sending any data to POST request")
+        data = None
+        if(store == 1):
+            data = input("Enter data as key value pairs")
+            data = json.loads(data)
```

Comment:
You can try loading the data inside of a `try..except` block.

Reply:
You were saying to implement ```try ... except ```  while converting str to json right?

Reply:
Yes. Because, if the JSON is invalid, it will throw the error to the screen. Instead, we can use our custom error function to let the user know about the error